### ISSUE TITLE
Ubuntu 19.04 won't register appimage

### ISSUE THREAD

**Issue by paranoidfactoid** [TheAssassin/AppImageLauncher#220]
Running Ubuntu 19.04. Installed both appimagelauncher 1.3.1 bionic and appimage launcher 1.4.0beta 2. It seems to start the daemon but doesn't pop up an integration window. And when I try to install an appimage manually from Nautilus, it starts the appimage but doesn't copy it from ~/Downloads to ~/Applications. Here's what it complains about in syslog:

> Aug  1 12:04:48 bukowski appimagelauncherfs[19134]: Registered new AppImage: "/home/foo/Downloads/krita-4.2.4-x86_64.appimage" (ID: 0010)
Aug  1 12:04:48 bukowski appimagelauncherfs[19134]: Error: could not find registered AppImage: /0010.AppImage.home
Aug  1 12:04:48 bukowski appimagelauncherfs[19134]: Error: could not find registered AppImage: /0010.AppImage.config
Aug  1 12:04:48 bukowski tracker-miner-f[3210]: Could not process 'file:///home/foo/%2328980472': Error when getting information for file “/home/foo/#28980472”: No such file or directory
Aug  1 12:04:49 bukowski org.gnome.Shell.desktop[2655]: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x500000e specified for 0x5000019.

**Comment by TheAssassin** [comment]
Have you tried to reboot? Have you changed any settings?

**Comment by paranoidfactoid** [comment]
Have I tried to reboot a linux box?!?! After installing AppimageLauncher?

**Comment by TheAssassin** [comment]
Well, don't ask me why, normally the installation works without any issues, but some systems which have been modified by the users are problematic there. Rebooting restarts all related services.

You have to reboot Linux systems every couple days anyway to update the kernel and close attack vectors.

**Comment by TheCodeSharman** [comment]
Can confirm. Running on Pop OS (Based on Ubuntu 19.04) - I get the same error in syslog (yes I rebooted just in case):
Aug 25 02:23:32 laptop appimagelauncherfs[1806]: Registered new AppImage: "/home/msharman/Tools/PrusaSlicer-2.0.0+linux64-201905201652.Ap
pImage" (ID: 0000)
Aug 25 02:23:32 laptop appimagelauncherfs[1806]: Error: could not find registered AppImage: /0000.AppImage.home
Aug 25 02:23:32 laptop appimagelauncherfs[1806]: Error: could not find registered AppImage: /0000.AppImage.config

**Comment by TheAssassin** [comment]
@TheCodeSharman your issue is related to #132. I'm not entirely sure what Prusa's slicer is trying to find in the portable home/config dirs...

**Comment by TheCodeSharman** [comment]
As a test I tried to download Subsurface and I get the same thing with Subsurface as well, also not registering like Prusa Slicer and giving the same error in syslog:
Aug 25 02:45:18 laptop appimagelauncherfs[1806]: Registered new AppImage: "/home/msharman/Downloads/Subsurface-4.9.1-x86_64.AppImage" (ID: 0003)
Aug 25 02:45:18 laptop appimagelauncherfs[1806]: Error: could not find registered AppImage: /0003.AppImage.home
Aug 25 02:45:18 laptop appimagelauncherfs[1806]: Error: could not find registered AppImage: /0003.AppImage.config

**Comment by TheAssassin** [comment]
These "errors" are just the runtime looking for potential directories, so called portable home/config. They look bad, but they're not "errors" in the sense that they'd prevent an AppImage from launching.

In fact, they prove the opposite: if these messages show up, there's proof that the runtime *within* the AppImage is running and doing stuff.

**Comment by TheCodeSharman** [comment]
Yes it runs but I thought it should integrate with the system - e.g. appear in menus? There is no evidence of this? I thought that what AppImageLauncher would do - add a icon so I can search for it via the GNOME shell interface rather than having to navigate to the folder with the AppImage installed and double click. Apologies if this isn't what AppImageLauncher does?

**Comment by TheCodeSharman** [comment]
Ah my bad - I didn't have the "Ask weather to move the AppImage into the applications directory" box checked in the AppImageLauncherSettings - that's why the integration didn't happen. Since I had already moved the AppImage myself I turned that off then wondered why there was no integration happening. Perhaps the wording needs to be clearer for new users?

**Comment by TheAssassin** [comment]
@TheCodeSharman I think we should just add more tooltips to the whole dialog. Also I think we need more translators. These are pretty hard to find, unfortunately...

**Comment by TheCodeSharman** [comment]
Hmm. I know its only a very minor issue but if I got confused, there are probably others who will too.

I think what I was looking for is an option to "Integrate" without moving the AppImage - which I realise now currently doesn't exist. Since "move the AppImage" is more specific than "integrate", I read it literally and was confused by this option also turning off the integration feature.

So my choice of wording would be "Ask whether to integrate the AppImage". The word "integrate" is used in the run/integrate dialogue box so this would be consistent and refers to the whole process of moving the AppImage and adding the menu icons. 

If there was a separate checkbox to disable to the "move to applications directory" functionality then this would make sense.

The addition of an option to not the move the file would be so I can move it where I want it on a case by case rather than all AppImages into one directory, the main use case would be to have different directories for different applications for instance or simply leave it where it is.

Can't help you with translators unfortunately, I'm a native English speaker and like too many in Australia we only speak one language, although they did try to teach me Indonesian in high school, which I guess makes a sort of geographic sense. Also I did try to learn a few words of German when I was last in the EU - but I'm far from being able to translate I'm afraid:-(
